src/app/app.component.ts

```diff
@@ -8,4 +11,23 @@ import {Component, NgModule} from '@angular/core';
 
 export class AppComponent {
   title = 'app';
+  weatherData:any = [];
+
+  constructor(private service:WeatherServiceService) {  }
+
+  ngOnInit(){
+    this.getData();
+  }
+
+  getData(){
```

Comment:
need to add a test for this method, testing that it calls the service 3 times & pushes data into weatherData

(i'd also remove the console.logs)